src/main/java/org/commcare/formplayer/objects/SerializableFormSession.java

```diff
@@ -11,24 +12,60 @@
 @Entity
 @Table(name = "formplayer_sessions")
 public class SerializableFormSession implements Serializable{
+    @Id @GeneratedValue(generator="system-uuid")
+    @GenericGenerator(name="system-uuid", strategy = "uuid")
```

Comment:
ID creation is now handled by the DB layer. This has the advantage of the DB layer knowing when an object is new vs exists which means it doesn't have to do a DB query to check (as we used to do here: https://github.com/dimagi/formplayer/blob/master/src/main/java/org/commcare/formplayer/repo/impl/PostgresFormSessionRepo.java#L128)